lib/algos/list/Extension.js

```diff
@@ -23,6 +23,32 @@ class Extension {
         this.keys = 0;
     }
 
+    /**
+     * Filters-out non-requested optional fields from the value. This function
+     * shall be applied on any value that is to be returned as part of the
+     * result of a listing extension.
+     *
+     * @param {Object} value - The JSON value of a listing item
+     *
+     * @return {Object} The value without the filtered-out fields.
+     */
+    filterContent(value) {
```

Comment:
minor: renaming to `trimContents` would be more precise I think

Reply:
ok for me